src/configs/latest.ts

```diff
@@ -47,6 +47,13 @@ export const rules = {
     }],
     "typeof-compare": true,
     "unified-signatures": true,
+
+    // added in v4.4
+    "arrow-return-shorthand": true,
+    "no-unnecessary-initializer": true,
+    "no-misused-new ": true,
+    "prefer-method-signature": true,
```

Comment:
let's remove this too; I think it's a little too opinionated and would fail a lot of our builds